railties/lib/rails/generators/app_base.rb

```diff
@@ -92,7 +92,7 @@ def self.add_shared_options_for(name)
                                            desc: "Set up the #{name} with Gemfile pointing to your Rails checkout"
 
         class_option :edge,                type: :boolean, default: nil,
-                                           desc: "Set up the #{name} with Gemfile pointing to Rails repository"
+                                           desc: "Set up the #{name} with Gemfile pointing to Rails repository stable branch"
```

Comment:
This is not always true. When it is inside a pre release it points to the main branch.

Reply:
Yes, but prereleases are also not necessarily stable either, are they? What would you call a `x-y-stable` branch instead of stable? I can update to add this caveat, but I think the term "edge" has always meant rails/main to me. Maybe we should just rename the option instead.

Reply:
Would it help to just name the branch it'll use?

Reply:
edge always had this implementation. It is edge in your current Rails series. But I agree with Matthew that we should just name the branch in the help.

Reply:
Ah, yeah that just clicked for me. Each stable branch is exclusive to a Rails version so we can just hardcode the name.